elliott/elliottlib/util.py

```diff
@@ -25,6 +26,8 @@
 now = datetime.datetime.now()
 YMD = '%Y-%b-%d'
 LOGGER = get_logger(__name__)
+GIT_REPO_BRANCH = "main"
```

Comment:
i don't think having this as a constant adds value .. , it is okay to hardcode it since it's an exceptional behavior

Reply:
thanks - updated in https://github.com/openshift-eng/art-tools/pull/1573/commits/544a92c5f8588db85ac8199fc945d3c181eb2b3c